template/.github/workflows/preview.yml

```diff
@@ -0,0 +1,27 @@
+name: Netlify Preview
+on:
+  push:
+    branches-ignore: ['main', 'master']
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+
+      - name: Install modules
+        run: yarn
+
+      - name: Build
+        run: yarn build
+
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          publish-dir: './build'
+          deploy-message: "Preview deployed from GitHub Actions"
```

Comment:
Should we change to using PR's title instead? So we can know which one is deployed on Netlify 🤔 
```suggestion
          deploy-message: ${{ github.event.pull_request.title }}
```

Reply:
But I think we can not use that variable since this will be triggered from `push` event

Reply:
I will check tomorrow, using the `on: pull_request` does not seems to re-trigger when adding new commits... 🤔 
And indeed, the variable only works with `on: pull_request` ;-)

Reply:
Working, I've concatenated the PR number with the PR title so it is easier to find it back in GitHub ;-)
![image](https://user-images.githubusercontent.com/77609814/162142776-01029b58-d22f-4189-b621-bad5b4e92b11.png)

Reply:
Done in 22a957a ✅